README.md

```diff
@@ -12,9 +12,84 @@ composer req bref/symfony-bridge
 
 ## Usage
 
-Update your Kernel.php
+You only need to one one small change To quickly setup Symfony to work with Bref.
 
 ```diff
+// src/Kernel.php
+
+namespace App;
+
+use App\Repository\Test\TestRepository;
++ Bref\SymfonyBridge\BrefKernel;
+use Symfony\Bundle\FrameworkBundle\Kernel\MicroKernelTrait;
+use Symfony\Component\Config\Loader\LoaderInterface;
+use Symfony\Component\Config\Resource\FileResource;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+-use Symfony\Component\HttpKernel\Kernel as BaseKernel;
+use Symfony\Component\Routing\RouteCollectionBuilder;
+
 - class Kernel extends BaseKernel
 + class Kernel extends BrefKernel
 ```
+
+Now you are up and running.
+
+## Optimize first request
+
+The first HTTP request that hits your application after you deployed a new version
+will use a cold cache directory. Symfony now spends time building thc cache. It may
+take everything between 1-20 seconds depending on the complexity of the application.
+
+Technically this happens whenever your application run on a new Lambda. That could
+be when you get a lot more traffic so AWS increases the resources or when AWS just
+decides to kill the lambda function (or server) that you are currently on. It is
+normal that this happens at least a handful of times every day.
+
+To optimize the first request, one must deploy the application with a warm cache.
+In a simple application it means that the deploy script looks something like this:
+
+```bash
+# Install dependencies
+composer install --classmap-authoritative --no-dev --no-scripts
+
+# Warmup the cache
+bin/console cache:warmup --env=prod
+
+# Disable use of Dotenv component
+echo "<?php return [];" > .env.local.php
```

Comment:
Does that make a noticable difference? Reading a php files vs reading a text file?

Reply:
On lambda I want to read from the environment variables. That is why I make sure the `.env.local.php` is empty.

Reply:
ahh, you point wasn't that that is a performance optimization but that you don't leak local environment variables to lambda. Got it.

Reply:
I updated the text a little to show that the interesting thing in this script is the `cache:warmup`